WordPressAuthenticator/Unified Auth/View Related/VerifyEmail/VerifyEmailViewController.swift

```diff
@@ -0,0 +1,224 @@
+import UIKit
+
+final class VerifyEmailViewController: LoginViewController {
+
+    // MARK: Properties
```

Comment:
super nit: just to be consistent in this file
```suggestion
    // MARK: - Properties
```

Reply:
Done in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/655/commits/c7856c23243e2c1d87d3e81455f05104e1a8f4ab